tests/e2e/tidbcluster/serial.go

```diff
@@ -589,6 +589,245 @@ var _ = ginkgo.Describe("[Serial]", func() {
 		})
 	})
 
+	ginkgo.Describe("Canary Deploy Operator from 1.1.10 to latest", func() {
+		var oa *tests.OperatorActions
+		var ocfg *tests.OperatorConfig
+		var operatorVersion string
+
+		ginkgo.BeforeEach(func() {
+			operatorVersion = "v1.1.10"
+			ocfg = &tests.OperatorConfig{
+				Namespace:       ns,
+				ReleaseName:     "operator",
+				Tag:             operatorVersion,
+				Image:           fmt.Sprintf("pingcap/tidb-operator:%s", operatorVersion),
+				ImagePullPolicy: v1.PullIfNotPresent,
+			}
+			oa = tests.NewOperatorActions(cli, c, asCli, aggrCli, apiExtCli, tests.DefaultPollInterval, ocfg, e2econfig.TestConfig, nil, fw, f)
+			ginkgo.By("Installing CRDs")
+			oa.DeployReleasedCRDOrDie(operatorVersion)
+			ginkgo.By("Installing tidb-operator")
+			oa.CleanOperatorOrDie(ocfg)
+			oa.DeployOperatorOrDie(ocfg)
+		})
+
+		ginkgo.AfterEach(func() {
+			ginkgo.By("Uninstall tidb-operator")
+			oa.CleanOperatorOrDie(ocfg)
+			ginkgo.By("Uninstalling CRDs")
+			tests.CleanReleasedCRDOrDie(operatorVersion)
+		})
+
+		ginkgo.It("Deploy TidbCluster and check the result", func() {
+			ginkgo.By(fmt.Sprintf("deploy original tc %q", utilimage.TiDBV4Prev))
+			tcName := "tidbcluster"
```

Comment:
```suggestion
			tcName := "tidbcluster1"
```

Reply:
There is already a case with the same name, and we'd better use a different name to avoid potential issues.